internal/app/grpcclient/impl.go

```diff
@@ -113,24 +115,224 @@ func (i *invoker) unary(
 }
 
 func (i *invoker) serverStream(
-	_ context.Context,
-	_ *v1alpha1.ClientCompatRequest,
+	ctx context.Context,
+	ccr *v1alpha1.ClientCompatRequest,
```

Comment:
![image](https://github.com/connectrpc/conformance/assets/2035234/c6ab767f-a34c-41f9-8c7f-3c577d1577af)

😹

Reply:
Lol I had the same thought.